### PR TITLE
Use bucketSsmKey for static assets

### PR DESCRIPTION
**Pull request by bryophyta**
## What does this change?

Uses `bucketSsmKey` field to look up the destination for static assets, as per DevX recommendations now that it's available for this resource type (see this [comment on a previous attempt to implement it](https://github.com/guardian/frontend/pull/25708#discussion_r1030592952) and [DCR issue #7098](https://github.com/guardian/dotcom-rendering/issues/7098)).

### CODE REVIEW

**Review by sunnyclimber456 — APPROVED** [review]

**Comment by bryophyta** [comment]
Thanks for the review! I'm reluctant to deploy last thing on a Friday so will leave this for now. I'm away until Thursday next week and will merge then (though if there's an urgent need then anyone else can feel free to take it on!) I haven't tested it in CODE yet, although I've checked the parameter store and @ioannakok and I did test the code for the other buckets, which was more-or-less identical.

**Pull request merged**
